doc/sphinx/practical-tools/utilities.rst

```diff
@@ -310,6 +316,16 @@ file timing data:
        (which are frequently noise); lexicographic sorting forces an order on
        files which take effectively no time to compile.
 
+       If you prefer a different sorting order, you can pass
+       ``TIMING_SORT_BY=absolute`` to sort by the total time taken, or
+       ``TIMING_SORT_BY=diff`` to sort by the signed difference in
+       time.
+
+    .. note::
+       Just like ``pretty-timed``, this table defaults to using user
```

Comment:
```suggestion
       Just like ``pretty-timed``, this table shows user
```

Reply:
This change causes the text to miss something important, imo.  It's not just showing user times, it's also basing the computed timing differences (and the sorting) on user times.

Reply:
Then ignore it.